player/scripting.c

```diff
@@ -242,14 +242,18 @@ static void load_builtin_script(struct MPContext *mpctx, int slot, bool enable,
 {
     mp_assert(slot < MP_ARRAY_SIZE(mpctx->builtin_script_ids));
     int64_t *pid = &mpctx->builtin_script_ids[slot];
-    if (*pid > 0 && !mp_client_id_exists(mpctx, *pid))
+    if (*pid > 0 && !mp_client_id_exists(mpctx, *pid)) {
+        MP_DBG(mpctx, "Assuming script %s has exited (client dead)\n", fname);
         *pid = 0; // died
+    }
     if ((*pid > 0) != enable) {
         if (enable) {
             *pid = mp_load_script(mpctx, fname);
         } else {
             char *name = mp_tprintf(22, "@%"PRIi64, *pid);
+            MP_DBG(mpctx, "Unloading script %s (disabled by option)\n", fname);
```

Comment:
Wouldn't be better to add `MP_TRACE` in `mp_client_send_event`, so we can see events in more generic way?

Reply:
I think that's a level below this, so there's justification for having both. Don't plan to add it however.